builtin/game/chatcommands.lua

```diff
@@ -985,3 +985,40 @@ core.register_chatcommand("clearinv", {
 		end
 	end,
 })
+
+local function handle_kill_command(murderer, victim)
```

Comment:
Dramatic language, how about 'killer' and 'killed'? Killing is not always a case of murderer and victim.

Reply:
to play the devil's advocate, it's a lot harder to confuse the two accidently when calling the function or editing it than with "killer" and "killed".

Reply:
Yes, there's probably a better word for 'killed'.

Reply:
murdered, eliminated, massacred, assassinated, wiped out, lynched and many more
`caller` (or `user`) and `target` could be softer names but I feel that's not necessary as the current variable names describe very well to whom they belong.

Reply:
You just don't have any sense of humour. :-(

Reply:
Hm well i can see the humour, but the words are not technical and neutral as in all other code.
Also, 'Killer' is an obvious choice because it matches the 'kill' function, just like 'digger' in 'dig' functions and 'placer' in 'place' functions.
'victim' may be ok, can't find a better word.